uvm/api/com/untangle/uvm/util/Constants.java

```diff
@@ -31,4 +31,8 @@ public class Constants {
     public static final String WEB_FILTER_EVENT_RGX = "*WebFilterEvent*";
     public static final String APP_CONTROL_LOG_EVENT_RGX = "*ApplicationControlLogEvent*";
 
+    // Commonly used keys
```

Comment:
Why are we adding these? Where are they used?

Reply:
Moved to DeviceTableImpl